limacharlie/artifact.go

```diff
@@ -76,6 +86,48 @@ func (org Organization) ArtifactRuleDelete(ruleName ArtifactRuleName) error {
 	return nil
 }
 
+func (org Organization) GetUploadUrl(oid string) (string, error) {
+
+	resp := orgUrls{}
+	var request restRequest
+	request = makeDefaultRequest(&resp)
+
+	if err := org.client.reliableRequest(http.MethodGet, fmt.Sprintf("orgs/%s/url", org.GetOID()), request); err != nil {
+		fmt.Println("Error making GET request:", err)
+		return "", nil
+	}
+
+	logsUrl := resp.URL
+	return logsUrl["logs"], nil
```

Comment:
```suggestion
	return logsUrl["artifacts"], nil
```
The `logs` is there for back-compat, but we've since renamed that subsystem to `artifacts`.